test/WebJobs.Script.Tests.E2E/FunctionAppFixture.cs

```diff
@@ -124,16 +124,22 @@ public FunctionAppFixture()
             Telemetry.TrackEvent("RuntimeUpdate", new Dictionary<string, string> { { "runtimeSource", Settings.RuntimeExtensionPackageUrl } });
 
             string extensionFilePath = Path.ChangeExtension(Path.GetTempFileName(), "zip");
-
-            using (var client = new HttpClient())
+            Uri packageUri = new Uri(Settings.RuntimeExtensionPackageUrl);
```

Comment:
I was able to easily verify the above fix using these new test infrastructure modifications. You can now set the RuntimeExtensionPackageUrl to a local zip file.